concepticondata/conceptlists/Araujo-1996-289.tsv

```diff
@@ -0,0 +1,290 @@
+Araujo-1996-289-223	223	Louse	1392.0	LOUSE	Laus	piolho
+Araujo-1996-289-224	224	Chiggoe (Jigger Flea)	438.0	CHIGGOE (JIGGER FLEA)	Sandfloh	bicho de pé
+Araujo-1996-289-225	225	Leech	2273.0	LEECH	Blutegel	sanguessuga
+Araujo-1996-289-226	226	Tree	906.0	TREE	Baum	árvore
+Araujo-1996-289-227	227	Forest	420.0	FOREST	Wald	floresta, mata
+Araujo-1996-289-228	228	Wood	1803.0	WOOD	Holz	madeira
+Araujo-1996-289-229	229	Liber (Inner Bark)			Bast	entrecasca
+Araujo-1996-289-230	230	Bark	1204.0	BARK	Rinde	casca
+Araujo-1996-289-231	231	Thorn	124.0	THORN	Dorne	espinho
+Araujo-1996-289-232	232	Leaf	628.0	LEAF	Blatte	folha
+Araujo-1996-289-233	233	Flower	239.0	FLOWER	Blüte	inflorescência
+Araujo-1996-289-234	234	Fruit	1507.0	FRUIT	Frucht	fruta
+Araujo-1996-289-235	235	Root	670.0	ROOT	Wurzel	raiz
+Araujo-1996-289-236	236	Sapucaia (Lecythis Pisonis)			Sapucaya	sapucaia
+Araujo-1996-289-237	237	Ambaiba (Cecropia)			Ambaúba	ambaúba
+Araujo-1996-289-238	238	Pacara Tree (Enterolobium Contortisiliquum)			Timbó	timbó
+Araujo-1996-289-239	239	Taquaruçu (Chusquea Gaudichaudii)			Taquaruçú	taquaruçú
+Araujo-1996-289-240	240	Grass	606.0	GRASS	Gras	capim
+Araujo-1996-289-241	241	Manioc	927.0	MANIOC	Mandioka	mandioca
+Araujo-1996-289-242	242	Sweet Potato	159.0	SWEET POTATO	Batate	batata-doce
+Araujo-1996-289-243	243	Yam	410.0	YAM	Inhame	inhame
+Araujo-1996-289-244	244	Maize	506.0	MAIZE	Mais	milho
+Araujo-1996-289-245	245	Bean	832.0	BEAN	Bohne	feijão
+Araujo-1996-289-246	246	Groundnut (Peanut)	595.0	GROUNDNUT (PEANUT)	Manduvy	amendoim
+Araujo-1996-289-247	247	Pumpkin	864.0	PUMPKIN	Kürbis	moranga
+Araujo-1996-289-248	248	Gourd	411.0	GOURD	Abobara	abóbora
+Araujo-1996-289-249	249	Watermelon	2766.0	WATERMELON	Melancia	melancia
+Araujo-1996-289-250	250	Pepper	1385.0	PEPPER	Pfeffer	pimenta
+Araujo-1996-289-251	251	Cotton	1850.0	COTTON	Baumwolle	algodão
+Araujo-1996-289-252	252	Tobacco	974.0	TOBACCO	Tabak	tabaco
+Araujo-1996-289-253	253	Achiote	349.0	ACHIOTE	Urucú	urucum
+Araujo-1996-289-254	254	I	1209.0	I	ich	eu
+Araujo-1996-289-255	255	Thou	1215.0	THOU	du	tu
+Araujo-1996-289-256	256	He	1211.0	HE	er	ele
+Araujo-1996-289-257	257	We Two	2634.0	WE TWO	wir zwei	nós dois (ambos)
+Araujo-1996-289-258	258	It Is Mine			ist mein	é meu
+Araujo-1996-289-259	259	It Is Yours			ist dein	é teu
+Araujo-1996-289-260	260	It Is His			ist sein	é dele
+Araujo-1996-289-261	261	My Foot			mein Fuß	meu pé
+Araujo-1996-289-262	262	Your Foot			dein Fuß	teu pé
+Araujo-1996-289-263	263	His Foot			sein Fuß	pé dele
+Araujo-1996-289-264	264	His Foot Is Red			sein Fuß ist rot	o pé dele é vermelho
+Araujo-1996-289-265	265	Our Feet			unsre Füße	nossos pés
+Araujo-1996-289-266	266	My Hand			meine Hand	minha mão
+Araujo-1996-289-267	267	Your Hand			deine Hand	tua mão
+Araujo-1996-289-268	268	His Hand			seine Hand	mão dele
+Araujo-1996-289-269	269	Our Hands			unsre Hände	nossas mãos
+Araujo-1996-289-270	270	My Hair Is Straight			mein Haar ist schlicht	meu cabelo é liso
```

Comment:
I would use "glatt" instead of "schlicht". Also for the following examples.

Reply:
Since the original is German, we'd have to leave it.